tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "compilerOptions": {
+    "experimentalDecorators": true,
```

Comment:
This is to not get typescript warnings on the `@autobind` decorators. (VS Code uses typescript to generate JS warnings)